### ISSUE TITLE
Why "filter" instead of "filters"?

### ISSUE THREAD

**Issue by vbwx** [msprev/panzer#46]
Since I keep my panzer support files in my pandoc data directory (to avoid redundancies), I wonder if there is a specific reason why panzer only looks for filters in a directory named "filter" when pandoc has a "filters" directory by convention?

The same applies to "template" vs "templates" but I've been successfully using "templates" as directory name so far.

**Comment by msprev** [comment]
Good point. This functionality of searching in these locations by default in pandoc did not exist (or at least I had not seen documentation on it) when panzer was written. 

I'll change panzer to look in both `templates` and `template`, etc.

The rationale for panzer using singular forms ('filter', 'template') was because I was understanding these terms to be used as verbs in the imperative mood rather than as collective nouns.

**Comment by vbwx** [comment]
Thanks, that will make panzer even more versatile.

I wanted to create a PR for this but I don't really know any Python yet 😞
